resources/additional-topics/modularity.markdown

```diff
@@ -993,7 +993,7 @@ If we break the tower, by giving it an impossible promise to keep, e.g. changing
 the name of the directory in tier 3 to something that cannot be created2, then
 tier 3 will fail and the output looks like this:
 
-``console
+```console
```

Comment:
artisanally crafted code blocks look OK to me.

heh some good wonks in there, quadruple backtick